src/spec/PromiseSpec.ts

```diff
@@ -1,6 +1,6 @@
 import * as ex from '@excalibur';
 
-describe('A promise', () => {
+xdescribe('A promise', () => {
```

Comment:
Were these tests failing as a result of the deprecation?

Reply:
I'll figure this out before we merge, seems to be something related to the `@obsolete()` some sort of dependency issue

Reply:
@jedeen Thanks for asking about this, when retargeting to es2015 classes the decorators no longer worked!!